examples/themes/netflix-theme.html

```diff
@@ -61,6 +61,16 @@
     height: 80px;
     padding-bottom: 20px;
     margin: 0 3px;
+
+    --media-button-icon-transform: scale(1.2);
+    --media-button-icon-transition: transform .2s ease;
```

Comment:
Unrelated to this change: do we currently document all the `--media-*` vars somewhere? It might be time to do that

Reply:
Yeah, I agree. I was starting to thing through what the right strategy is. These are elements, so I'd love to keep things looking as much like styling a native element, and only use variables in special cases.

The rub being that custom elements don't give you a good way to style many things at once. If I want to change the background of all button controls, I have to name them all specifically, or provide a variable. Or another option would be to force a class name on those elements, but I'm not sure how well that would play with other things.